tests/core/pyspec/eth2spec/test/bellatrix/transition/test_transition.py

```diff
@@ -33,3 +36,31 @@ def test_sample_transition(state, fork_epoch, spec, post_spec, pre_tag, post_tag
 
     yield "blocks", blocks
     yield "post", state
+
+
+@with_fork_metas([
+    ForkMeta(pre_fork_name=pre, post_fork_name=post, fork_epoch=8) for pre, post in AFTER_BELLATRIX_PRE_POST_FORKS
+])
```

Comment:
The two tests look general and can be reused with future forks:

1. Merge `bellatrix/transition/test_transition.py` into `altair/transition/test_transition.py`
2. Replace `AFTER_BELLATRIX_PRE_POST_FORKS` with `ALL_PRE_POST_FORKS`.
3. `AFTER_BELLATRIX_PRE_POST_FORKS` can be removed totally.